ui-spacetimechart/src/utils/canvas.ts

```diff
@@ -139,8 +174,13 @@ export function drawAliasedDisc(
   { x: centerX, y: centerY }: Point,
   radius: number,
   [r, g, b]: RGBColor | RGBAColor,
-  drawOnBottom: boolean
+  drawOnBottom: boolean,
+  scalingRatio: number = 1
 ): void {
+  centerX = Math.round(centerX * scalingRatio);
+  centerY = Math.round(centerY * scalingRatio);
+  radius = Math.round(radius * scalingRatio);
+
   centerX = Math.round(centerX);
   centerY = Math.round(centerY);
   radius = Math.ceil(radius);
```

Comment:
These seem unnecessary now?

Reply:
I merged the two parts and restored the `Math.ceil` call.